app/src/main/java/com/example/home/User.java

```diff
@@ -0,0 +1,28 @@
+package com.example.home;
+
+
+import org.json.JSONObject;
+import org.parceler.Parcel;
+
+import java.util.ArrayList;
+@Parcel
+public class User {
+    public String userName;
```

Comment:
Attributes should be private, and you should give access to tthem through getters and setters.